docs/flank-output-investigation/flank_output_propositon.md

```diff
@@ -0,0 +1,101 @@
+# Flank output proposition
+
+## Android
+
+### Version section
+
+```bash
+
+version: local_snapshot
+revision: d5e7c76ab206373c6cdf0c1a25e2575f323e5929
+session id: 7a7ceda1-658e-461f-bd37-04f2b2a90a37
+
+```
+
+### Args section
+
+```bash
+<Diff of arguments>
+
+```
+
+### Dump shards
+
+```bash
+[Dump shards]
+  Saved 1 shards to /Users/adamfilipowicz/Repos/flank/results/2021-03-01_13-01-56.722225_lYrQ/android_shards.json
+  Uploading [android_shards.json] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+```
+
+### Uploading files
+
+```
+[Uploading files]
+
+  Uploading [app-debug.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [app-single-success-debug-androidTest.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [session_id.txt] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
```

Comment:
IMO uploading files looks too verbose comparing to gloud. There is a lot of places in the output where this same link occurs - `https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/`. 

The output could be more smooth and readable without that information duplicated in many places.

Reply:
Maybe we should go with
```
Uploading [matrix_ids.json] to GCloud..
```

or 
```
Uploading [matrix_ids.json] ...
```

WDYT?

Reply:
Or we could 1:1 gcloud output 

```

Uploading [../test_projects/android/apks/app-debug.apk] to Firebase Test Lab...

```

With part of the path to file WDYT?

Reply:
For files uploaded for testing:
```to Firebase Test Lab...```
For other files:
```to Gcloud ...```

is it ok ?

Reply:
I think yes

Reply:
let's include it in the next task for implementation